scripts/wrk2-prometheus.yaml.tmpl

```diff
@@ -24,6 +24,7 @@ spec:
       - name: tempfs
         emptyDir:
           medium: Memory
+      shareProcessNamespace: true
```

Comment:
This is not enough to kill the linkerd proxy. You will also need get rid of the `runAsUser` field and let this process run as root. For this to be allowed, you will need permissions set in the PSP. Like it is done in this PR: https://github.com/kinvolk/service-mesh-benchmark/pull/21

Reply:
Thanks, will update the PR.